site/en/workflows.md

```diff
@@ -774,10 +896,98 @@ This processor can be used to analyse the output of the OCR.
       <td>&nbsp;</td>
       <td>First input group should point to the ground truth.</td>
 	  <td><code>ocrd-dinglehopper -I OCR-D-GT,OCR-D-OCR -O OCR-D-EVAL</code></td>
+    </tr>
+	<tr>
+      <td>ocrd-cor-asv-ann-evaluate</td>
+      <td>
+      <p>
+      <code>
+      {"metric": "Levenshtein" (default), "NFC", "NFKC", "historic-latin"}
+      </code>
+      <code>
+      {"confusion": integer}
+      </code>
+      </p>	  
+	  </td>
+      <td>First input group should point to the ground truth. There is no output file group, it only uses logging. If you want to save the evaluation findings in a file, you could e.g. add `2> eval.txt` at the end of your command</td>
+	  <td><code>ocrd-cor-asv-ann-evaluate -I OCR-D-GT,OCR-D-OCR</code></td>
     </tr>
   </tbody>
 </table>
 
+## Format Conversion (Optional)
+
+OCR-D produces PAGE XML files which contain the recognized text as well as detailed
+information on the structure of the processed pages, the coordinates of the recognized
+elements etc. Optionally, the PAGE XML can be converted to a different output format.
+
+### Step 18: Format Conversion
+
+In this processing step the produced PAGE XML files can be converted to ALTO,
+PDF, hOCR or text files. Note that ALTO and hOCR can also be converted into
+different formats whereas the PDF version of PAGE XML OCR results is a widely
+accessible format that can be used as-is by expert and layman alike.
+
+#### Available processors
+
+<table class="processor-table">
+<thead>
+    <tr>
+      <th>Processor</th>
+      <th>Parameter</th>
+      <th>Remarks</th>
+      <th>Call</th>
+    </tr>
+
+</thead>
+<tbody>
+    <tr>
+      <td>ocrd-fileformat-transform</td>
+      <td><code>
+        {"from-to": "alto2.0 alto3.0"} 
+        <br># or {from-to: "alto2.0 alto3.1"}
+        <br># or {from-to: "alto2.0 hocr"}
+        <br># or {from-to: "alto2.1 alto3.0"}
+        <br># or {from-to: "alto2.1 alto3.1"}
+        <br># or {from-to: "alto2.1 hocr"}
+        <br># or {from-to: "alto page"}
+        <br># or {from-to: "alto text"}
+        <br># or {from-to: "gcv hocr"}
+        <br># or {from-to: "hocr alto2.0"}
+        <br># or {from-to: "hocr alto2.1"}
+        <br># or {from-to: "hocr text"}
+        <br># or {from-to: "page alto"}
+        <br># or {from-to: "page hocr"}
+        <br># or {from-to: "page text"}
+      </code>
+      </td>
+      <td>&nbsp;</td>
+      <td><code>ocrd-fileformat-transform -I OCR-D-OCR -O OCR-D-ALTO</code></td>
+    </tr>
+
```

Comment:
```suggestion
```

This newline might be responsible for the broken table rendering. (I guess the Markdown engine tries to pick it up.)